action.yml

```diff
@@ -20,6 +20,10 @@ inputs:
     description: 'Use librepo for downloading packages for the output image. (can break if you are using an old bib image)'
     required: false
     default: false
+  rootfs:
+    description: 'Root filesystem type. Overrides the default from the source container. Supported values: ext4, xfs, btrfs'
+    required: false
+    default: ''
```

Comment:
Please could we default to creating an ISO, to prevent a breaking change on the action?  
I know of a few repos which use this to generate ISOs, so it would be good to not require them to make any changes.

Reply:
The current default value will exclude the arg `--rootfs=${input}` from being added to the build container, this should behave exactly as it previously did with the defaults for an ISO build.

Would you prefer `'default'` or similar instead of an empty-string to convey that the rootfs is not being overridden?

Kicked off a test ISO build: https://github.com/leftl/mucore/actions/runs/14014022125
It had the following inputs:
```
          bootc-image-builder-image: ghcr.io/centos-workstation/bootc-image-builder:latest
          use-librepo: true
          config-file: ./iso.toml
          image: ${{ env.IMAGE_REGISTRY }}:${{ env.DEFAULT_TAG }}
```

Reply:
Ah, apologies.  I must have got mixed up with the rootfs and type.  
That's why you don't review PRs after midnight 😆 

This looks great as-is.  Thank you so much!